web/components/notifications/notification-helpers.tsx

```diff
@@ -47,7 +51,7 @@ export function PrimaryNotificationLink(props: {
   }
   return (
     <span className="font-semibold transition-colors hover:text-indigo-500">
-      {truncateText(text, truncatedLength ?? 'xl')}
+      {truncatedLength ? truncateText(text, truncatedLength ?? 'xl') : text}
```

Comment:
I don't think the truncations in the middle of the text is helpful. I feel pretty strongly that it should only truncate text at the end, and I believe James agrees.